src/openforms/emails/api/serializers.py

```diff
@@ -0,0 +1,6 @@
+from rest_framework import serializers
+
+
+class ConfirmationEmailTemplateSerializer(serializers.Serializer):
```

Comment:
The reason was so we can delete the `ConfirmationEmailTemplate` if a dict without a subject or body filled in doesn't throw a validation error.

For example if a `ConfirmationEmailTemplate` exists and the user wants to delete it they'll need to remove the subject and content which then leaves a dict like
```
{
   'subject': '',
   'content': ''
}
```
but this would thrown an error if this were a model serializer.

I guess a comment would be good to put there explaining this ;)